test/test_base_providers.py

```diff
@@ -22,10 +22,16 @@ class TestBaseProvider(unittest.TestCase):
     """
 
     def setUp(self) -> None:
+        self._setUpWithEIBM("benchmark_EI_OECM_PC.json")
+
+    def setUpS3(self) -> None:
+        self._setUpWithEIBM("benchmark_EI_S3.json")
+
+    def _setUpWithEIBM(self, eibm_filename) -> None:
         self.root = os.path.dirname(os.path.abspath(__file__))
         self.company_json = os.path.join(self.root, "inputs", "json", "fundamental_data.json")
         self.benchmark_prod_json = os.path.join(self.root, "inputs", "json", "benchmark_production_OECM.json")
-        self.benchmark_EI_json = os.path.join(self.root, "inputs", "json", "benchmark_EI_OECM_PC.json")
+        self.benchmark_EI_json = os.path.join(self.root, "inputs", "json", eibm_filename)
```

Comment:
Later in this function, at new lines 48-49, the company data is re-parsed.  But the fundamental company data (in JSON format) specifies intensity and target projections only for S1S2 data.  With no S3 projections, the S3 budget cannot be calculated in the aggregation step.  I'm putting in a temporary fix copying the S1S2 data and declaring it as S3 data.  A better fix would be to put in actual data (both for S1S2 and S3).